package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tracing-middleware",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
a breaking change bump to `0.3.0` seems more appropriate, since the peer dependency bump on `opentracing` is also a breaking change